Fitness/Views/AllProRoutinesView.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  AllProRoutinesView.swift
+//  Fitness
+//
+//  Created by Cameron Hamidi on 3/21/19.
+//  Copyright © 2019 Keivan Shahida. All rights reserved.
+//
+
+import UIKit
+
+extension UIView {
+    func addRoundedCornersAndShadows(radius: CGFloat) {
+        layer.cornerRadius = radius
```

Comment:
Where's the shadow part?

Reply:
That function is not supposed to apply shadows, I will change the name